lib/dpl/provider/convox.rb

```diff
@@ -41,6 +41,12 @@ def push_app
         end
       end
 
+      private
+
+      def build_description
+        options[:description] || sha
```

Comment:
Will `sha` return the latest commit hash?

Reply:
If I understand [Provider#sha](https://github.com/plan3/dpl/blob/master/lib/dpl/provider.rb#L171) - yes.

Reply:
Oh yeah, you're right. I thought `sha` was some built-in Ruby mumbojumbo.